packages/dd-trace/test/setup/mocha.js

```diff
@@ -113,10 +125,19 @@ function withNamingSchema (
         })
       })
 
+      beforeEach(async () => {
+        await new Promise(resolve => setTimeout(resolve, 30))
```

Comment:
Between these two sleep statements it appears that we're slowing down every `withNamingSchema` test by 60ms?